metric-example/query-by-example.md

```diff
@@ -0,0 +1,516 @@
+```
+stock_price:last
+```
+If there is at least one non-null piece of data, the result will use the latest value or otherwise be empty.
+
+You can customize the behavior of last by specifying as a first parameter how many entries should be considered from the end of the series (default: all entries), and as a second parameter how many of the last data points should be aggregated together for the final result (default: one, only use the final datum), if you, e.g. want the average of the last three values instead:
+```
+stock_price:last(24,3)
+```
+# Other Functionality Related to Metric Selectors
+
+The parameters `from`, `to` and `resolution` are not part of the metric selector but apply to the whole query. Nonetheless, this section provides a high-level overview that will help you use them effectively.
+
+## Choosing Resolution and Time Frame
+
+`from` and `to` accept multiple date formats, including milliseconds since 1970 and human-readable dates in year-month-day order that may include time-of-day, as well as GMT offset. Additionally, a simple format for relative dates is accepted.
+
+The recommended way to format an absolute date known in advance is to fully specify it. For instance, the time of Nepali New Year in 2019 is identified with `2019-04-14T00:00:00+05:45` in the local time zone. If the bounds are calculated from within a script or in code, it is recommended to use a millisecond timestamp instead, e.g. `1554798800839`. Note that regardless of the format of the requested time frame, datetimes in responses will always be in UTC.
+
+Sometimes, especially when using the API interactively, it is desirable to use a time relative to the current instant in time when making the request. The API facilitates this by accepting the time anchor `now` with simple arithmetic on top. For instance, minutes (`m`), hours (`h`), weeks (`w`), months (`M`) or years (`y`) can be added or subtracted from `now` using `+` and `-` operators. Furthermore, we support an alignment operator `/` that zeroes parts of the date smaller than the specified unit. Some examples:
+
+| Expression | Meaning |
+|---|---|
+| `now-1w` | One week before now, same time-of-day as now |
+| `now/w` | The beginning of the current week (Monday, 00:00:00.00) |
+| `now/w-d` | Also beginning of the week, depending on who you ask (Sunday, 00:00:00.00) |
+| `now/M+5m` | Five minutes into the current month |
+
+Resolution accepts identical units, e.g. `M` or `1M` signifies one month of time between data points. Instead of directly specifying the resolution, a unit-less quantity can be specified that identifies a number of data points that should be equally distributed in the desired query time frame. The API will then choose the most coarse resolution available that will result in at least the desired amount of data points.

```

Comment:
```suggestion
Resolution accepts identical units, e.g. `M` or `1M` signifies one month of time between data points. Instead of directly specifying the resolution, a unit-less quantity can be specified that identifies a number of data points that should be equally distributed in the desired query timeframe. The API will then choose the most coarse resolution available that will result in at least the desired amount of data points.

```